word/pkg/client/client.go

```diff
@@ -0,0 +1,27 @@
+package client
+
+import (
+	"context"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	msgraphsdkgo "github.com/microsoftgraph/msgraph-sdk-go"
+
+	"github.com/gptscript-ai/tools/word/pkg/global"
+)
+
+// StaticTokenCredential is taken from https://github.com/gptscript-ai/mail-assistant/blob/10944805801bbb6f71eccefd1bea5f114fded164/pkg/mstoken/auth.go
```

Comment:
nit: remove comment.
Opinion: it's not relevant where this is copied from, just that it works.

Reply:
done